test/known_problems/should_fail/int_op.erl

```diff
@@ -0,0 +1,11 @@
+-module(int_op).
+-export([plusfloatany/2, divfloatany/2]).
+
+-spec plusfloatany(float(), any()) -> integer().
+plusfloatany(X, Y) ->
+    A = X + Y,
+    A.
+
```

Comment:
The function above passes for the same reason as `lambda_wrong_args.erl`. The root of the problem is that we don't do a good job with variables and match statements. Integer operators has nothing to do with the problem.
Perhaps we should have a separate file in `known_problems/should_fail` which just isolates the problem with variables.

Reply:
That seems like a good idea!